index.js

```diff
@@ -11,7 +11,11 @@ module.exports = function expressGAuth(options) {
     errorNoUser: (req, res, next) => res.send('<h1>Error logging in, no user!</h1>'),
     errorLogin: (req, res, next, err) => res.send('<h1>Login error!</h1>'),
     serializeUser: (user, done) => done(null, user),
-    deserializeUser: (user, done) => done(null, user)
+    deserializeUser: (user, done) => done(null, user),
+    googleAuthorizationParams: {
+      scope: ['profile', 'email'],
+      prompt: 'select_account'
+    }
```

Comment:
`Object.assign` is used for combining parameters and defaults -> caller cannot override deep object properties individually.

Eg. `gauth({ googleAuthorizationParams: { hostedDomain: 'example.com' } })` will unset `scope` and `prompt`.

I think we should do a deep merge here?

Reply:
This was intentional but I can see why merging could make more sense.

Though with merging I suppose only way to remove a property from defaults is to pass undefined as value and then we'd pick pairs with non-undefined value.

Reply:
Discussed this separately with @miksu and @rjoonas and we concluded that simple override is probably better option here.